CardScan/Classes/API.swift

```diff
@@ -135,20 +136,22 @@ public struct Api {
         apiCall(endpoint: endpoint, parameters: apiParameters, completion: completion)
     }
     
-    static public func donwloadLatestModel(signedUrl: String, completion: @escaping ApiCompletion) {
+    @available(iOS 11.0, *)
+    static public func downloadAndCompileLatestModel(signedUrl: String, completion: @escaping ApiCompletion) {
         guard let url = URL(string: signedUrl) else {
             DispatchQueue.main.async { completion(nil, apiUrlNotSet) }
             return
         }
         
         let session = URLSession(configuration: configuration())
         session.downloadTask(with: url) { (location: URL?, response: URLResponse?, error: Error?) in
-            guard let location = location else {
+            guard let location = location, let compiledUrl = try? MLModel.compileModel(at: location) else {
                 DispatchQueue.main.async { completion(nil, defaultError) }
                 return
             }
+            
             DispatchQueue.main.async {
-                completion(["mlmodel_url": location], nil)
+                completion(["compiled_model_url": compiledUrl], nil)
```

Comment:
will changing this break any existing code? I couldn't find anything referencing this string, is it used anywhere?

Reply:
this function is used in model updates in cardverify

Reply:
Got it, so there will be a corresponding change in CardVerify [here](https://github.com/getbouncer/cardverify-ios/blob/7d0918d7681ac66f2d62476a26c0d84b457a2a38/CardVerify/Classes/DynamicModelController.swift#L86)? makes sense, although this is breaking. Do we have any way of indicating which versions of CardVerify are compatible with which versions of CardScan?

Reply:
yes that is correct. We indicate compatibility in the cardverify podspec file, where cardverify's version dependency of cardscan is stated.